deploy/ray-operator/README.md

```diff
@@ -0,0 +1,135 @@
+# Ray-Operator Documentation
+
+[Kubernetes](https://kubernetes.io/) is an open-source system for automating deployment, scaling, and management of containerized applications. It is playing an important role in the next generation cloud-infra. 
+
+So running [Ray](https://ray.readthedocs.io/en/latest/) on Kubernetes will make Ray more acceptable to the industry.  
+
+The [CR(Custom Resource defined in Kubernetes)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and the controller we propose allow users to customize almost all the configurations of Kubernetes and Ray. 
```

Comment:
This file looks like it's just cut and paste from the doc? Could you spend some time refining it a bit to describe what the operations supported by the operator are and why it's useful to users?

Reply:
I've modified this part to show some features for user why they use this. You can view again.